src/testdir/test_window_cmd.vim

```diff
@@ -1789,4 +1789,35 @@ function Test_nosplitscroll_cmdwin_cursor_position()
   %bwipeout!
   set splitscroll&
 endfunction
+
+" No scroll when aucmd_win is opened.
+function Test_nosplitscroll_aucmdwin()
+  set nosplitscroll
+
+  call setline(1, range(1, &lines))
+  norm Gzz
+  let top = line('w0')
+  let id = popup_atcursor('test', {})
+  call setbufvar(winbufnr(id), '&syntax', 'c')
+  call assert_equal(top, line('w0'))
+  call popup_close(id)
+
+  set splitscroll&
+endfunc
```

Comment:
@brammool if you could wait with this patch I will try to find a test that also works for neovim.

Reply:
Ready now.